### DISCUSSION TITLE
How do I set the sharing capability?

### DISCUSSION THREAD

**Discussion by plamber** [pnp/pnpcore#1045] [Q&A] [answered]
Hello,
I haven't found a way to configure the sharing capability for a site. The only part that I found was reading the property `SharingCapability` using the `GetSiteCollectionManager`.

If there is not such a capability, I would love to add it to the SDK. I just need to know how you would call the method and where you would put it.

Thank you,
Patrick

**Comment by jansenbe** [comment, marked as answer]
@plamber : see https://pnp.github.io/pnpcore/using-the-sdk/admin-sharepoint-tenant.html#setting-tenant-properties. First you get the properties of the site, update the needed values in the model and call one of the `Update` methods

**Reply by plamber** [reply]
Thank you. Missed it.
